docs/user/dql/functions.rst

```diff
@@ -2730,6 +2730,95 @@ Another example to show how to set custom values for the optional parameters::
     +------------+
 
 
+MATCHQUERY
+-----
+
+Description
+>>>>>>>>>>>
+
+``matchquery(field_expression, query_expression[, option=<option_value>]*)``
+
+The matchquery function maps to the match query used in search engine, to return the documents that match a provided text, number, date or boolean value with a given field. This is alternate syntax for the `match`_ function. Available parameters include:
```

Comment:
Please mention that this was added for backwards compatibility.  See the match_phrase section:
https://github.com/Bit-Quill/opensearch-project-sql/blob/726ddbd006cfbb9c7e9f19231e5706f48de5e6ee/docs/user/dql/functions.rst#match-phrase

Reply:
The way it's done here, they don't create a separate section for `matchphrase` and `match_phrase`. Is this preferred and should I remove the entry I've made for `matchquery` here? Or would you prefer to keep the separate section for `matchquery` and just mention that it was added for compatibility somewhere in the text?

Reply:
just add a sentence like:
```
For backward compatibility, matchphrase is also supported and mapped to match_phrase query as well.
```

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/163/commits/cbccd339c0bf812c36e3e9ee2a88e638457cf9e4